src/frontend/src/eth/components/send/EthSendTokenWizard.svelte

```diff
@@ -289,13 +291,9 @@
 		>
 			<svelte:fragment slot="cancel">
 				{#if formCancelAction === 'back'}
-					<button type="button" class="secondary block flex-1" on:click={back}
-						>{$i18n.core.text.back}</button
-					>
+					<ButtonBack type="button" on:click={back} />
```

Comment:
Is there a use case where the back or cancel button `submit` a form? We can maybe set those per default to `button` ?

Reply:
there is no case, BUT, the default is `submit` when it is not given, for the built-in` <button>` component. So we can either keep the same behaviour of the built-in, or I can check the code for the usage (but I am very afraid I can lose track of something)

Reply:
Not sure we are speaking about the same idea. I meant for example modifying `ButtonCancel` as following:

```
<script lang="ts">
	import Button from '$lib/components/ui/Button.svelte';
	import { i18n } from '$lib/stores/i18n.store';
	
	// REMOVE THIS
	export let type: 'submit' | 'reset' | 'button' = 'submit';
	
	export let disabled = false;
</script>

// MAKE type = button here
<Button colorStyle="secondary" type="button" {disabled} on:click>
	{$i18n.core.text.cancel}
</Button>
```

Reply:
Yes yes, indeed, but my point is that the instances where it was used without `type` provided, for example:

```html
<button class="secondary block flex-1" on:click={() => dispatch('icCancel')}>
  {$i18n.core.text.cancel}
</button>
```

 means that they were automatically `submit` type (that is the default if nothing is provided).
 
So to maintain the same behavior and avoid any disruptions, we need to omit it too, let it default to `submit`.
 
Otherwise, I should check all instances mentioned to check if one of them MUST explicitly have `submit` type. I can do that too, no worries

Reply:
Good point. Let's pass this PR and then modify the buttons in another PR? I feel like "cancel" and "back" should not be "submit" maybe?

Reply:
right! will do!

Reply:
Done with:

- PR #2544